core/src/components/img/img.tsx

```diff
@@ -77,6 +87,7 @@ export class Img implements ComponentInterface {
         src={this.loadSrc}
         alt={this.alt}
         decoding="async"
+        onError={this.loadError}
```

Comment:
wondering why we can't just use onError directly

Reply:
That error trigger even before the image start loading as there is no `src`.

Reply:
@pickfire  I think manu means removing  `onError={this.loadError}` and instead using the listener:

```
@Listen('error')
onImgError(ev: Event) {
 // Your code from loadError
}
```

Reply:
I tested that but it does not seem to work.
```typescript
  /** Emitted when img failed to load */
  @Event() ionError!: EventEmitter<void>;

  @Listen('error')
  onImgError() {
    this.ionError.emit();
  }
  ...
  render() {
    return (
      <img
        src={this.loadSrc}
        alt={this.alt}
        decoding="async"
        // onError={this.loadError}
      />
    );
  }
```

Reply:
I believe this is because the error event does not bubble: https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onerror